.gitignore

```diff
@@ -55,6 +58,5 @@ docs/_build/
 # PyBuilder
 target/
 
-# example
-examples/static
```

Comment:
Wouldn't we need these directories plus `examples/*.db`? I understand that this is not in `cookiecutter` because not all modules use db in the example app but this one does @krzysztof ?

Reply:
Ah you're right